Hotfix release-20200914 (#19)
tidbcloud/dbaas-docs: release-notes.md

This page lists the release notes of [TiDB Cloud](https://pingcap.com/products/tidbcloud).

## September 14, 2020

* Fix monitoring metrics by adding the `region` label

> * Fix Non-HTAP cluster can not be scaled issue

Reviewer: ```suggestion
* Fix the issue that non-HTAP clusters cannot be scaled
```

This page lists the release notes of [TiDB Cloud](https://pingcap.com/products/tidbcloud).

## September 14, 2020

> * Fix monitoring metrics by adding `region` label

Reviewer: ```suggestion
* Fix monitoring metrics by adding the `region` label
```
